inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -88,6 +88,7 @@ Engine::Engine() {
 }
 
 Engine::~Engine() {
+    ExecutorManager::getInstance()->clear("CPU");
```

Comment:
Do we really have "CPU" key? [getIdleCPUStreamsExecutor](https://github.com/openvinotoolkit/openvino/blob/d18073260bc742d7bf14d262d6919a1b660e2b61/inference-engine/src/inference_engine/threading/ie_executor_manager.cpp#L25) is called only for "CPUStreamsExecutor" and "CPUCallbackExecutor": https://github.com/openvinotoolkit/openvino/blob/a5e24977880deeab3b3476e50bf17efba80876b9/inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp#L228-L241

Reply:
See:
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp#L228-L231

Reply:
@ilya-lavrenov, yeah but I cannot find a place where `getExecutor` adds an entry to `cpuStreamsExecutors`.

Reply:
Oh, thanks! I just tracked `cpuStreamsExecutors` but missed `executors`